src/main/kotlin/no/nav/dagpenger/journalføring/ferdigstill/JournalPostApi.kt

```diff
@@ -5,27 +5,59 @@ import com.github.kittinunf.fuel.core.extensions.authentication
 import com.github.kittinunf.fuel.core.extensions.jsonBody
 import com.github.kittinunf.fuel.httpPatch
 import com.github.kittinunf.fuel.httpPut
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
 import mu.KotlinLogging
 import no.nav.dagpenger.oidc.OidcClient
 
 private val logger = KotlinLogging.logger {}
 
 internal interface JournalPostApi {
     fun ferdigstill(journalPostId: String)
-    fun oppdater(journalPostId: String, sak: Sak)
+    fun oppdater(journalPostId: String, jp: OppdaterJournalPostPayload)
 }
 
+internal data class OppdaterJournalPostPayload(
+    val avsenderMottaker: Avsender,
+    val bruker: Bruker,
+    val tittel: String,
+    val sak: Sak,
+    val dokumenter: List<Dokument>,
+    val behandlingstema: String = "ab0001",
+    val tema: String = "DAG",
+    val journalfoerendeEnhet: String = "9999"
+)
+
+enum class SaksType {
+    GENERELL_SAK,
+    FAGSAK
+}
+
+internal data class Sak(val saksType: SaksType, val fagsakId: String?, val fagsaksystem: String?)
+internal data class Avsender(val navn: String)
+internal data class Bruker(val id: String, val idType: String = "FNR")
+internal data class Dokument(val dokumentinfoId: String, val brevkode: String, val tittel: String)
```

Comment:
Tror ikke det er vits å oppdatere brevkode. Jf. oppdaterJournalpost:

>Kode som sier noe om dokumentets innhold og oppbygning.
For inngående dokumenter kan brevkoden være en NAV-skjemaID f.eks. "NAV 14-05.09" eller en SED-id.
Brevkode skal ikke settes for ustrukturert, uklassifisert dokumentasjon, f.eks. brukeropplastede vedlegg.

For selve hoveddokumentet og alt som skal ha brevkode, er dette satt på forhånd.